api/solarperformanceinsight_api/models.py

```diff
@@ -268,7 +286,7 @@ class Inverter(BaseModel):
         "", title="Make & Model", description="Make and model of the inverter"
     )
     arrays: List[PVArray] = Field(
-        ..., description="PV arrays that are connected to this inverter"
+        ..., description="Names of the PV arrays that are connected to this inverter"
```

Comment:
```suggestion
        ..., description="List of PV arrays that are connected to this inverter"
```

This is actually a list of PVArray objects, not just the name. A full example of what a system looks like might be helpful https://github.com/SolarPerformanceInsight/solarperformanceinsight/blob/c041ac5babbc648df5b6e951e841b8da607cda6d/api/solarperformanceinsight_api/models.py#L8-L58